content/applications/finance/expenses.rst

```diff
@@ -448,89 +544,110 @@ to select all reports in the list. Next, click on the :guilabel:`⚙️ Action (
+:guilabel:`Re-Invoicing Expenses`:
 
-- :guilabel:`No`: Expense product will not be re-invoiced.
-- :guilabel:`At cost`: Expense product will invoice expenses at their real cost.
-- :guilabel:`At sales price`: Expense product will invoice the price set on the sale order.
+- :guilabel:`None`: Expense category will not be re-invoiced.
+- :guilabel:`At cost`: Expense category will invoice expenses at their real cost.
+- :guilabel:`At sales price`: Expense category will invoice the price set on the sale order.
 
 Create an expense
 -----------------
 
 First, when :ref:`creating a new expense <expenses/new>`, the correct information needs to be
-entered in order to re-invoice a customer. Select the :guilabel:`Customer to Reinvoice` from the
-drop-down menu. Next, select the :guilabel:`Analytic Account` the expense will be posted to.
+entered in order to re-invoice a customer. Select the *sales order* the expense will appear on in
+the :guilabel:`Customer to Reinvoice` section, from the drop-down menu. Next, select the
+:guilabel:`Analytic Account` the expense will be posted to. After the expense(s) are created, the
+expense report needs to be :ref:`created <expenses/report>` and :ref:`submitted <expenses/submit>`
+as usual.
 
 .. image:: expenses/reinvoice-expense.png
    :align: center
    :alt: Ensure the customer to be invoiced is called out on the expense.
 
-Create a quote and sales order
-------------------------------
-
-In the :menuselection:`Sales` app, create a quote for the customer being invoiced, listing the
-expense products. First, click :guilabel:`Create` to create a new quotation. Next, select the
-:guilabel:`Customer` being invoiced for the expenses from the drop-down menu.
-
-In the :guilabel:`Order Lines` tab, click :guilabel:`Add a product`. In the :guilabel:`Product`
-field, select the first item being invoiced from the drop-down menu, or type in the product name.
-Then, update the :guilabel:`Quantity`, the :guilabel:`Delivered` quantity, and the :guilabel:`Unit
-Price` if needed. Repeat this for all products being invoiced. When all the products have been added
-to the quote, click :guilabel:`Confirm` and the quotation becomes a sales order.
-
-.. image:: expenses/expenses-salesorder.png
-   :align: center
-   :alt: Create and confirm the sales order with the expenses listed as products.
-
-Once the quote turns into a sales order, a :guilabel:`Delivered` column appears. The delivered
-quantity must be updated for each item. Click on the `0.000` field for each product, and enter the
-delivered quantity. When all delivered quantities have been entered, click :guilabel:`Save`.
+.. important::
+   Selecting a :guilabel:`Customer to Reinvoice` when creating an expense is critical, since this
+   is what causes automatic invoicing after an expense report is approved. The :guilabel:`Customer
+   to Reinvoice` field can be modified *until an expense report is* **approved**, then the field is
+   no longer able to be modified.
```

Comment:
Early line breaks
```suggestion
   Selecting a :guilabel:`Customer to Reinvoice` when creating an expense is critical, since this is
   what causes automatic invoicing after an expense report is approved. The :guilabel:`Customer to
   Reinvoice` field can be modified *until an expense report is* **approved**, then the field is no
   longer able to be modified.
```